src/schemas/json/pre-commit-config.json

```diff
@@ -3,6 +3,19 @@
     "title": "JSON schema for .pre-commit-config.yaml",
     "type": "object",
     "properties": {
+        "ci": {
+            "$comment": "Settings specific to https://pre-commit.ci/",
+            "type": "object",
+            "properties": {
```

Comment:
Should add all of the configuration properties if adding this high-level property: https://pre-commit.ci/#configuration